tests/playbooks/test_minimal.yaml

```diff
@@ -2,9 +2,8 @@
   import_playbook: _before_adoption.yaml
 
 - name: Adoption
-  hosts: "{{ adoption_host | default('localhost') }}"
+  hosts: local
```

Comment:
Are you sure this is correct?

Reply:
I'm personally not sure about "local", but the same is used in other playbooks. It works in those playbooks.

Reply:
The adoption_host was actually introduced by Pini here: https://github.com/openstack-k8s-operators/data-plane-adoption/commit/6e44f379ac3d9a40876beb81eea606f69a222331

But i'm not sure if it's still needed.